packages/core/src/components/overlay/overlay.tsx

```diff
@@ -84,6 +84,11 @@ export interface IBackdropProps {
     backdropProps?: React.HTMLProps<HTMLDivElement>;
 
     /**
+     * Whether underlying content should be scrollable while the backdrop is visible.
+     */
+    backgroundScrollingDisabled?: boolean;
```

Comment:
Hmm, I feel like we already have a prop for that: `hasBackdrop`. If it's set to `true`, you have a backdrop behind and you _can't_ scroll. If it's `false`, you don't have a background behind and you _can_ scroll.

Enabling this extra `backgroundScrollingDisabled` prop means that we think there's a use case for having a backdrop behind and being able to scroll what's beneath it. I'm skeptical about that but open to it.

What about `canOutsideClickClose` found in `Overlay`, and `isModal` found in `Popover`? I feel like there's a lot of repetition going on... Maybe `isModal` is what we want instead of `backgroundScrollingDisabled`.